README.md

```diff
@@ -10,6 +10,8 @@ Packages:
 - [Honeycomb Kubernetes Agent](./charts/honeycomb)
 - [Honeycomb Network Agent](./charts/network-agent)
 - [Honeycomb Refinery](./charts/refinery)
+- [Honeycomb Telemetry Pipeline](./charts/htp) (based on Bindplane)
+- [Honeycomb Observability Pipeline](./charts/observability-pipeline) (experimental)
```

Comment:
```suggestion
```

Lets not add this one yet as it is not intended to be used yet. No need to direct anyone too it.

Reply:
lgtm per slack